bmeg_app/views/home_view.py

```diff
@@ -25,68 +25,89 @@
 # Page
 #######
 NAME=i18n.t('app.config.tabname_widget_home')
+
+cluster_card = dbc.Card(
```

Comment:
Is there a way to get this set of cards built by responding to a config file?

Reply:
my first thought would be to create a function that inputs header and body string, then returns the full `dbc.Card()`